DevSkim-DotNet/Microsoft.DevSkim/Ruleset.cs

```diff
@@ -123,6 +236,11 @@ public void AddFile(string filename, string? tag = null)
         public void AddRange(IEnumerable<Rule> collection)
         {
             _rules.AddRange(collection);
+            foreach(var rule in collection.Select(DevSkimRuleToConvertedOatRule))
+            {
+                if (rule != null)
```

Comment:
Curious under what conditions the collection might have a null rule i.e. can we avoid this test?